### PR TITLE
Initial commit for neofs-adm utility

### PR DESCRIPTION
**Pull request by alexvanin**
Build with `make bin/neofs-adm`.

```
$ ./bin/neofs-adm config init 
Initial config file saved to /home/alexvanin/.neofs/adm/config.yml

$ cat /home/alexvanin/.neofs/adm/config.yml 
rpc-endpoint: https://neo.rpc.node:30333
alphabet-wallets: /home/alexvanin/.neofs/adm/alphabet-wallets
network:
  max_object_size: 67108864
  epoch_duration: 240
# if credentials section is omitted, then neofs-adm will require manual password input
credentials:
  az: password
  buky: password
  vedi: password
  glagoli: password
  dobro: password
  yest: password
  zhivete: password

$ ./bin/neofs-adm config init --path ./.neofs-adm.yaml
Initial config file saved to ./.neofs-adm.yaml
```

### CODE REVIEW

**Review by realloc — APPROVED** [review]

File: cmd/neofs-adm/internal/modules/config/config.go [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+	if err != nil {
+		return err
+	}
+
+	_, err = f.WriteString(configText)
+	if err != nil {
+		return fmt.Errorf("writing to %s: %w", configPath, err)
+	}
+
+	err = f.Sync()
```
Comment (cthulhu-rider) [inline cmd/neofs-adm/internal/modules/config/config.go:64]: Can we open file with `os.O_SYNC`?

File: cmd/neofs-adm/internal/modules/config/config.go [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+	if configPath != "" {
+		return configPath, nil
+	}
+
+	return defaultConfigPath()
+}
+
+func defaultConfigPath() (string, error) {
+	home, err := homedir.Dir()
```
Comment (cthulhu-rider) [inline cmd/neofs-adm/internal/modules/config/config.go:88]: Why don't we use `os.UserHomeDir()`?

File: cmd/neofs-adm/internal/modules/config/config.go [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+	"path"
+	"path/filepath"
+	"text/template"
+
+	"github.com/mitchellh/go-homedir"
+	"github.com/nspcc-dev/neofs-node/pkg/innerring"
+	"github.com/spf13/cobra"
+)
+
+type (
```
Comment (carpawell) [inline cmd/neofs-adm/internal/modules/config/config.go:16]: why is it in brackets?

File: cmd/neofs-adm/internal/modules/root.go [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+}
+
+func Execute() error {
+	return rootCmd.Execute()
+}
+
+func entryPoint(cmd *cobra.Command, args []string) error {
+	printVersion, err := cmd.Flags().GetBool("version")
+	if err == nil && printVersion {
+		cmd.Printf("neofs-adm %s\n", misc.Version)
```
Comment (carpawell) [inline cmd/neofs-adm/internal/modules/root.go:41]: in https://github.com/nspcc-dev/neofs-node/commit/eadc3a4de94cfdfd621bdf61daeccdcfbd4505c1 i made all "version outputs" be in the same format, wouldn't it be better to make the same here? or to change the format in `cli`/`IR`/`SN`

File: cmd/neofs-adm/internal/modules/config/root.go [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+import (
+	"github.com/spf13/cobra"
+)
+
+const configPathFlag = "path"
+
+var (
+	// RootCmd is a root command of config section.
+	RootCmd = &cobra.Command{
```
Comment (carpawell) [inline cmd/neofs-adm/internal/modules/config/root.go:11]: is it ok to name both cobra command in that cli as "root"?

**Review by carpawell — APPROVED** [review]

File: cmd/neofs-adm/internal/modules/config/root.go (hunk 5 above)
Comment (alexvanin) [reply cmd/neofs-adm/internal/modules/config/root.go:11]: It is. They are stored in different packages for this purpose.

**Review by cthulhu-rider — APPROVED** [review]

File: cmd/neofs-adm/internal/modules/config/config.go [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+}
+
+const configTxtTemplate = `rpc-endpoint: {{ .Endpoint}}
+alphabet-wallets: {{ .AlphabetDir}}
+network:
+  max_object_size: {{ .MaxObjectSize}}
+  epoch_duration: {{ .EpochDuration}}
+# if credentials section is omitted, then neofs-adm will require manual password input
+credentials:{{ range.Glagolitics}}
+  {{.}}: password{{end}}
```
Comment (fyrchik) [inline cmd/neofs-adm/internal/modules/config/config.go:30]: What's the command for generating empty section?

**Review by fyrchik — APPROVED** [review]

File: cmd/neofs-adm/internal/modules/config/config.go (hunk 6 above)
Comment (alexvanin) [reply cmd/neofs-adm/internal/modules/config/config.go:30]: There is no such command. Default config will always have `credential` section with seven alphabet keys.
Administrator is supposed to change this config later and this section can be removed by administrator (that what the comment above for).

**Pull request merged**